src/status_im2/common/nickname_drawer/view.cljs

```diff
@@ -0,0 +1,83 @@
+(ns status-im2.common.nickname-drawer.view
+  (:require [clojure.string :as string]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icons]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.common.nickname-drawer.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn empty-nickname?
```

Comment:
That's what I thought were your intentions. If you named the function `valid-nickname?` I wouldn't have made this comment, but since the name of the function is just a rewording of `string/blank?`, then it's sort of unnecessary indirection.